checkbox-ng/plainbox/impl/session/test_resume.py

```diff
@@ -2031,6 +2031,41 @@ def test_simple_session(self):
         # as specific tests for restoring results are written elsewhere
         self.assertEqual(session.job_state_map[job.id].result.outcome, "pass")
 
+    def test_simple_session_with_rejected(self):
+        """
+        verify that _restore_SessionState_jobs_and_results() works when
+        faced with a representation of a simple session (no generated jobs
+        or anything "exotic").
```

Comment:
This is the same comment as `test_simple_session`, I guess you may want to update it to mention the rejected jobs.

Reply:
done